File: C/divisibilidade.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

int divisor, dividendo;

int main()
{
    printf("");

    while(true){
        printf("DIVIDENDO: ");
        scanf("%d", &dividendo);
        printf("DIVISOR: ");
        scanf("%d", &divisor);
        
        if(dividendo % divisor == 0) printf("É divisível!");
        if(dividendo % divisor != 0) printf("Não é divisível!");
        printf("\n------------------------------\n");
    }
    
    return 0;
}

File: Python/divisibilidade.py
isRunning = True

while isRunning:
    print("#####[ É DIVISÍVEL OU NÃO? ]#####")
    dividendo = input("Dividendo: ")
    divisor = input("Divisor: ")
    
    if int(dividendo) == 0 or int(divisor) == 0:
        break
    
    resultado = int(dividendo) % int(divisor)

    if resultado == 0:
        print("É DIVISÍVEL! RESTO = 0!\n\n")
    else:
        print("NÃO É DIVISÍVEL! RESTO = " + resultado + "\n\n")